rocketpy/utilities.py

```diff
@@ -15,6 +15,20 @@
 from .AeroSurfaces import TrapezoidalFins
 
 
+def except_negative(method):

```

Comment:
I believe this functions should goes into the `tools.py` file.
I'm opened to discussions